src/config/index.js

```diff
@@ -24,6 +24,7 @@ const configuration = {
   NODE_ENV: process.env.NODE_ENV,
   CUSTOMER_SUPPORT_EMAIL: 'customersuccess@edx.org',
   TABLEAU_URL: process.env.TABLEAU_URL,
+  ENTERPRISE_LEARNER_PORTAL_HOSTNAME: process.env.ENTERPRISE_LEARNER_PORTAL_HOSTNAME,
```

Comment:
just a process question: so for prod env, how does this work? do we set en env var in the webpack.prod.config.js at some point or do we directly set an env var for prod somewhere outside of source control (hopefully not this latter one)?

Reply:
@iloveagent57 assures me that this env variable is already provided in our prod environment. So, we test on stage, and that's also why I made sure this variable exists before using it.

Reply:
ah ok that is the part I was not clear on, looks like we have a way to set the env vars for prod, was curious about that one ... but not an issue for this PR